src/styles/blocks/stars.scss

```diff
@@ -0,0 +1,24 @@
+.stars {
+  display: flex;
+
+  &__star {
+    margin-right: 4px;
+    width: 16px;
+    height: 16px;
+    background-image: url(./../images/star.svg);
+    background-repeat: no-repeat;
+    background-position: center;
+  }
+
+  & > :last-child {
+    margin-right: 0;
+  }
+
+  &--1 > :nth-child(-n+1),
+  &--2 > :nth-child(-n+2),
+  &--3 > :nth-child(-n+3),
+  &--4 > :nth-child(-n+4),
+  &--5 > :nth-child(-n+5) {
+    background-image: url(./../images/star-active.svg);
```

Comment:
Use the correct path for the star-active.svg image. Replace './../images/star-active.svg' with '../images/star-active.svg'.